### PR TITLE
XIVY-14102 vite package core and editor to make it esm conform

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -17,14 +17,17 @@
...
-  "module": "lib/index",
-  "source": "src/index",
+  "types": "lib/index.d.ts",
+  "main": "lib/core.js",
+  "module": "lib/core.js",
   "scripts": {
     "clean": "rimraf lib tsconfig.tsbuildinfo",
     "build": "tsc --build && tscp",
-    "package": "tsc && tscp",
+    "package": "npm run clean && vite build && npm run build",
```
Comment (ivy-cst) [inline packages/core/package.json:30]: vite and tsc needed?

**Review by ivy-cst — APPROVED** [review]
Cool if it helps :+1:

File: packages/core/package.json (hunk 1 above)
Comment (ivy-lmu) [reply packages/core/package.json:30]: not sure ^^inspired by: 
https://github.com/axonivy/ui-components/blob/f5f7d2b84388f763cbfb8933de1235855e795024/packages/components/package.json#L73C3-L74C11

**Comment by ivy-lmu** [comment]
unfortunately it does not work properly because the css files are not packaged

**Comment by ivy-lli** [comment]
I do not understand why this should be needed...

**Pull request merged**